src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -356,13 +359,14 @@ public static CloseableHttpClient buildHttpClient(
       connectionManager.setMaxTotal(maxConnections);
       connectionManager.setDefaultMaxPerRoute(maxConnectionsPerRoute);
 
+      String userAgentIdentifier = key != null ? key.getUserAgentIdentifier() : "";
       HttpClientBuilder httpClientBuilder =
           HttpClientBuilder.create()
               .setConnectionManager(connectionManager)
               // Support JVM proxy settings
               .useSystemProperties()
               .setRedirectStrategy(new DefaultRedirectStrategy())
-              .setUserAgent(buildUserAgent()) // needed for Okta
+              .setUserAgent(buildUserAgent(userAgentIdentifier)) // needed for Okta
```

Comment:
Rename variable `userAgentIdentifier` to `userAgentSuffix` everywhere

Reply:
Done